app/main/routes.py

```diff
@@ -556,17 +559,42 @@ def search_transferring_body(_id: uuid.UUID):
             {3: {"search_terms": query if query == "," else display_terms}}
         )
 
-        fuzzy_search_query = build_fuzzy_search_transferring_body_query(
-            query,
-            transferring_body_id=_id,
-            sorting_orders=sorting_orders,
+        open_search = OpenSearch(
+            hosts=current_app.config.get("OPEN_SEARCH_HOST"),
+            http_auth=current_app.config.get("OPEN_SEARCH_HTTP_AUTH"),
+            use_ssl=True,
+            verify_certs=False,
+            connection_class=RequestsHttpConnection,
         )
 
-        search_results = fuzzy_search_query.paginate(
-            page=page, per_page=per_page
-        )
+        dsl_query = {
+            "query": {
+                "bool": {
+                    "must": [
+                        {
+                            "multi_match": {
+                                "query": query,
+                                "fields": ["*"],
+                            }
+                        }
+                    ],
+                    # "filter": [
```

Comment:
we do need this uncommented and pass in the transferring_body_id since this route is searching within a transferring body